media/public/include/IMediaPipelineCapabilities.h

```diff
@@ -96,6 +96,16 @@ class IMediaPipelineCapabilities
      * @retval true if supported.
      */
     virtual bool isMimeTypeSupported(const std::string &mimeType) = 0;
+
+    /**
+     * @brief  Has any gstreamer sink or decoder got a named property
+     *
+     * @param[in] mediaType : media source type
```

Comment:
We should put in the interface about when mediaType is unknown, like you did in the proto file.